packages/jmix-react-antd/src/ui/form/controls/CheckboxField.tsx

```diff
@@ -0,0 +1,22 @@
+import {Checkbox, CheckboxProps} from "antd";
+import * as React from "react";
+import {JmixFormFieldWrapper} from "./base/JmixFormFieldWrapper";
+import {JmixFormFieldProps} from "./base/JmixFormFieldProps";
+
+export function CheckboxField({
+  entityName, propertyName, formItemProps, value, onChange, ...rest
+}: JmixFormFieldProps & CheckboxProps) {
+  return (
+    <JmixFormFieldWrapper entityName={entityName}
+                          propertyName={propertyName}
+                          formItemProps={{valuePropName: "checked", ...formItemProps}}
+                          renderField={(isReadOnly) => (
+                            <Checkbox value={value}
+                                      onChange={onChange}
```

Comment:
I'm starting to think that maybe we don't need to pass `value` and `onChange` explicitly and maybe we don't need to have them in `JmixFormFieldProps`. `Form.Item` will "magically" pass them (in this specific component it will pass value as `checked` prop as you've specified `valuePropName`). We shouldn't need to ever pass them manually.

Reply:
We still can specify `value` and `onChange` in `FormFields`, `value` will be ignored, `onChange` can be used as listener

Reply:
In this case I suggest to change it to:

```
<Checkbox checked={value}
```

Otherwise `value` will be ignored.

Reply:
Passed value `<CheckboxField value={value}/>` will be ignored in that case too, because `checked` property "binded" to form field values, it's ok i think. In other components like `<TextField>` we can't change value through the prop `value` too